training/index.html

```diff
@@ -0,0 +1,273 @@
+<article class="training_container">
+  <section class="training">
+    <h2 class="text-2xl caps">General AGRC Resources</h2>
+    <ul class="fa-ul">
+      <li><span class="fa-li text-orange"><i class="fas fa-map-marker-alt"></i></span><a href="{% link index.html %}">AGRC</a>: The AGRC website is Utah’s GIS data clearinghouse.</li>
+      <li><span class="fa-li text-orange"><i class="fas fa-map-marker-alt"></i></span><a href="{% link sgid/open-sgid/index.md %}">Open SGID</a>: The Open SGID is a publicly available PostGIS database containing State Geographic Information Database (SGID) (i.e., all Utah vector data).</li>
+      <li><span class="fa-li text-orange"><i class="fas fa-map-marker-alt"></i></span><a href="https://opendata.gis.utah.gov" rel="noopener noreferrer" target="_blank">Open Data</a>: AGRC’s Open Data site is the SGID served as ArcGIS Online feature services.</li>
+      <li><span class="fa-li text-orange"><i class="fas fa-map-marker-alt"></i></span><a href="https://raster.utah.gov" rel="noopener noreferrer" target="_blank">Raster Discovery Application</a>: AGRC’s Raster Discovery Application allows users to search and download aerial photography, elevation data, and USGS topographic maps for a user-defined area of interest.</li>
+    </ul>
+    <h3 class="text-xl ml-1">AGRC’s Discover Imagery and Base Map Server Resources</h3>
+    <ul class="fa-ul">
+      <li><span class="fa-li text-orange"><i class="fas fa-map-marker-alt"></i></span><a href="{% link discover/index.html %}">AGRC’s Base Map and Imagery Services</a>: Discover provides imagery and base maps services in Open Geospatial Consortium (OGC) standard Web Map Tile Service (WMTS) and Web Map Service (WMS) in the Web Mercator WGS84 projection (wkid: 3857). This page provides general background information on Discover and explains how to <a href="{% link discover/index.html %}#sign-up">sign up for access to Discover</a>.</li>
+      <li><span class="fa-li text-orange"><i class="fas fa-map-marker-alt"></i></span><a href="{% link discover/resources/index.md %}">Discover Server Resources and Information</a>: Once users have signed up for Discover, they can check out this page to learn everything there is to know about consuming the services available from Discover.</li>
+    </ul>
+  </section>
+  <section class="training-next">
+    <h2 class="text-2xl caps">Training</h2>
+  </section>
+  <section class="training-left">
+    <h3 class="text-xl"><a href="https://esri.com/training" rel="noopener noreferrer" target="_blank">Online Esri Training</a></h3>
+    <p> Esri has many online courses that are included in the annual software maintenance cost. They also have several basic free courses that are a good introduction to GIS and ArcMap/ ArcGIS Pro.</p>
+    <h3 class="text-xl"><a href="https://www.esri.com/training/" rel="noopener noreferrer" target="_blank">Free Esri Courses</a></h3>
```

Comment:
**Suggestion:** Remove link, which is redundant with the link in row 32. Anna is also removing this link from the pdf.

Reply:
if you press the `+/-` you can make the suggestion much easier for me. Read about it on [number 6](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)

Reply:
resolving because you added a new comment